src/node_buffer.cc

```diff
@@ -465,11 +475,21 @@ MaybeLocal<Object> New(Environment* env,
     }
   }
 
-  Local<ArrayBuffer> ab =
-      ArrayBuffer::New(env->isolate(),
-                       data,
-                       length,
-                       ArrayBufferCreationMode::kInternalized);
+auto callback = [](void* data, size_t length, void* deleter_data){
```

Comment:
I think the indentation is a bit off here? i.e.

```suggestion
  auto callback = [](void* data, size_t length, void* deleter_data){
```

Reply:
fixed